app/controllers/follows_controller.rb

```diff
@@ -0,0 +1,23 @@
+class FollowsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_question, only: [:create, :destroy]
+
+  authorize_resource
```

Comment:
No, you don't) Even in `destroy` action. `authorize_resurce`'s callback is called before `destroy`, there is no `@follow`.

It's better to remove `authorize_resource` and use `authorize ...` within action. Less magic – better.

Btw, you should also check for double-subscription in `#create`.